generators/generator-base.js

```diff
@@ -262,29 +270,41 @@ Generator.prototype.getAllInstalledLanguages = function () {
  * get all the languages supported by JHipster
  */
```

Comment:
we broke the filesystemanyway means modules ahave to change as well so this is also ok I guess, just let all hipsters with module know that if they are using this method